src/pass.h

```diff
@@ -20,6 +20,7 @@
 #include <functional>
 
 #include "mixed_arena.h"
+#include "passes/passes.h"
```

Comment:
Why is this here?

Reply:
This was originally intended to access `createGenerateStackIRPass`, but is no longer needed.